emcc.py

```diff
@@ -2336,6 +2336,9 @@ def check_memory_setting(setting):
     # always does.
     settings.DEFAULT_LIBRARY_FUNCS_TO_INCLUDE += ['$callRuntimeCallbacks']
 
+  if settings.EXIT_RUNTIME and not settings.STANDALONE_WASM:
```

Comment:
Why not in standalone mode?

Reply:
In standalone mode we implement `_start` which is in charge of doing the enture `ctors + main + atexit` sequence so we only need to export that one function. 

Hopefully we can do this for non-standalone mode one day too.